recipes/modern-cpp-kafka/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+# pylint: skip-file
```

Comment:
```suggestion
```

This is only allowed for v1-only tests, which should use `test_v1_****` pattern. Below I suggest the [required changes](https://github.com/conan-io/conan-center-index/blob/master/docs/v2_linter.md#import-tools-from-conan)

Reply:
I'm wondering whether there's anything missed, -- if I changed the `test_package` folder to `test_v1_package` (together with the `bin_path` for the test), the test package would not be found! 🤐

Reply:
> I'm wondering whether there's anything missed, -- if I changed the `test_package` folder to `test_v1_package` (together with the `bin_path` for the test), the test package would not be found! 🤐

Oh, I see, -- since it's not `ready for Conan v2` (I tried to migrate, but failed and have to give up), we don't need the `pylint` line either. Let me have a try. Thanks!😀